qiling/os/windows/fncc.py

```diff
@@ -32,7 +33,6 @@ def _x86_get_params_by_index(ql, index):
 
 
 def _x8664_get_params_by_index(ql, index):
-    #self.ql = ql
     reg_list = [UC_X86_REG_RCX, UC_X86_REG_RDX, UC_X86_REG_R8, UC_X86_REG_R9]
```

Comment:
u might be able to use  "rcx" and things here. so can reduce the import of unicorn everywhere